config/base_boxes.yaml

```diff
@@ -76,3 +76,13 @@ boxes:
     box: centos7
     <<: *install_shell
     options: --install-type=devel
+
+  centos6-foreman:
+    box: centos6
+    <<: *install_shell
+    options: --install-type=foreman
+
+  centos7-foreman:
+    box: centos7
+    <<: *install_shell
+    options: --install-type=foreman
```

Comment:
It's also not a big deal to continue to maintain the YAML with the list of versions, but would it be possible with the Vagrant file to make this more dynamic? By that I mean, when Foreman 1.10 comes out, just being able to say `vagrant up centos7-foreman-1.10` without needing to do any changes.  Just a thought, it could be done later.